### PR TITLE
Final priv edits

### PR DESCRIPTION
**Pull request by paul-grassi**

### CODE REVIEW

File: sp800-63c/sec9_privacy.md [hunk 1]
```diff
@@ -33,9 +32,9 @@ Due to the many components of digital authentication, it is important for the SA
...
 
-Note that even with the use of blinding technologies, it MAY still be possible for a blinded party to infer protected subscriber information through released attribute data or metadata such as analysis of timestamps, attribute bundle sizes, or attribute signer information. 
+It may still be possible for a blinded party to infer protected subscriber information through released attribute data or metadata such as analysis of timestamps, attribute bundle sizes, or attribute signer information. The IdP could consider additional privacy-enhancing approaches to reduce the risk of revealing identifying information of the entities participating in the federation.
```
Comment (jricher) [inline sp800-63c/sec9_privacy.md]: We should restore the "Even with the use of blinding technologies, it may still..." as this is the hinge of the caveat.

Comment (paul-grassi) [reply sp800-63c/sec9_privacy.md]: This is in the Blinding Section so I felt that caveat was redundant.  I don't feel strong enough to debate this one, so if you still think it should be there, I can put it back.

**Review by nbl-nist — APPROVED** [review]

**Comment by nbl-nist** [comment]
LGTM

File: sp800-63c/sec9_privacy.md (hunk 1 above)
Comment (jricher) [reply sp800-63c/sec9_privacy.md]: I think that it helps connect the thoughts together, so I'd like to see it back in. I agree with the removal of the normative MAY (I think that was an editing mistake in the first place).

Comment (paul-grassi) [reply sp800-63c/sec9_privacy.md]: Just committed the change.

**Review by jricher — APPROVED** [review]

**Pull request merged**